### PR TITLE
chore: Merge kyma tests into integration

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Merge kyma tests into integration

**Related issue(s)**

https://github.com/kyma-project/busola/issues/3298

**Definition of done**

- [x] The PR's title starts with one of the following prefixes:
  - feat: A new feature
  - fix: A bug fix
  - docs: Documentation only changes
  - refactor: A code change that neither fixes a bug nor adds a feature
  - test: Adding tests
  - chore: Maintainance changes to the build process or auxiliary tools, libraries, workflows, etc.
- [x] Related issues are linked. To link internal trackers, use the issue IDs like `backlog#4567`
- [x] Explain clearly why you created the PR and what changes it introduces
- [x] All necessary steps are delivered, for example, tests, documentation, merging

### CODE REVIEW

File: tests/integration/cypress.config.js [hunk 1]
```diff
@@ -77,6 +78,21 @@ module.exports = defineConfig({
       'tests/namespace/test-limit-ranges.spec.js',
       'tests/namespace/test-resource-quotas.spec.js',
       'tests/namespace/z-run-after.spec.js',
+      // uncomment after frog fix
+      // 'tests/kyma-cluster/test-applications.spec.js',
+      // 'tests/kyma-cluster/test-modules-wizard.spec.js',
```
Comment (mrCherry97) [inline tests/integration/cypress.config.js:83]: Remove this, wizards are no longer in use

File: tests/integration/cypress.config.js [hunk 2]
```diff
@@ -77,6 +78,21 @@ module.exports = defineConfig({
...
+      'tests/kyma-namespace/a-run-before.spec.js',
+      'tests/kyma-namespace/test-certificates.spec.js',
+      'tests/kyma-namespace/test-issuers.spec.js',
+      'tests/kyma-namespace/test-dns-providers.spec.js',
+      'tests/kyma-namespace/test-hpa.spec.js',
+      'tests/kyma-namespace/test-oauth2.spec.js',
+      'tests/kyma-namespace/test-dns-entries.spec.js',
+      'tests/kyma-namespace/test-services.spec.js',
+      // 'tests/kyma-namespace/test-kyma.spec.js',
+      // 'tests/kyma-namespace/test-module-templates.spec.js',
```
Comment (mrCherry97) [inline tests/integration/cypress.config.js:94]: Remove this, this view is removed

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: tests/integration/cypress.config.js (hunk 2 above)
Comment (dbadura) [reply tests/integration/cypress.config.js]: Removed

File: tests/integration/cypress.config.js (hunk 1 above)
Comment (dbadura) [reply tests/integration/cypress.config.js]: Removed

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**